src/_pyfuse3.py

```diff
@@ -308,9 +308,11 @@ def init(self):
         :manpage:`open(2)` manpage and defined in the `os` module (with the
         exception of ``O_CREAT``, ``O_EXCL``, ``O_NOCTTY`` and ``O_TRUNC``)
 
-        This method must return an integer file handle. The file handle will be
-        passed to the `read`, `write`, `flush`, `fsync` and `release` methods to
-        identify the open file.
+        This method must return a ``FileInfo`` instance. The `fh` field must
```

Comment:
This should be a single backquote so that it becomes a link in the HTML documentation.